README.md

```diff
@@ -97,13 +97,13 @@ cd lemmyBB
 LEMMYBB_VERSION=$(git describe --tag --always) cargo build --release
 ```
 
-Copy the nginx config into the sites-enabled folder and edit it to fit your setup
+Copy the nginx config into the sites-enabled folder
 
 ```
 cp docker/nginx.conf /etc/nginx/sites-enabled/lemmybb.conf
 ```
 
-replace the variable `lemmybb.com` with your domain for lemmybb
+and edit it to fit your setup by replacing the variable `lemmybb.com` with your domain for lemmybb
```

Comment:
This might be misleading, you dont need to edit the file but only run the sed command below, with your actual domain instead of lemmybb.com